avro-fastserde/src/main/java/com/linkedin/avro/fastserde/backport/ResolvingGrammarGenerator.java

```diff
@@ -448,6 +451,16 @@ private static Symbol mkEnumAdjust(List<String> wsymbols,
     return Symbol.enumAdjustAction(rsymbols.size(), adjustments);
   }
 
+  private static Symbol mkEnumAdjustWithDefault(Schema writer, Schema reader) {
+    Resolver.EnumAdjust e = (Resolver.EnumAdjust) Resolver.EnumAdjust.resolve(writer, reader, GenericData.get());
+    Object[] adjs = new Object[e.adjustments.length];
+    for (int i = 0; i < adjs.length; i++) {
+      adjs[i] = (0 <= e.adjustments[i] ? new Integer(e.adjustments[i])
```

Comment:
It is a little weird to write this way, so can you modify it to be `e.adjustments[i] >= 0` instead?

Reply:
I just copied the logic from Avro